src/components/form/_form.scss

```diff
@@ -182,7 +182,7 @@
   }
 
   .#{$prefix}--form__helper-text {
-    @include typescale('omega');
+    @include type-style('helper-text-01');
     font-style: italic;
     color: $text-02;
     line-height: 1.5;
```

Comment:
The height is still wrong should be `line-height: 1` (or 16px). Its actually built into the type style: 
![image](https://user-images.githubusercontent.com/11670886/53745243-42360900-3e64-11e9-8a3a-f9c827bcf17b.png)

Reply:
thanks for catching that, I updated the style rules and added helper text to the examples in the experimental docs